src/main/java/com/powsybl/sld/server/PositionDiagramLabelProvider.java

```diff
@@ -32,7 +29,7 @@
 /**
  * @author Ben Daamer ahmed<ahmed.bendaamer at rte-france.com>
  */
-public class PositionDiagramLabelProvider extends DefaultLabelProvider {
+public class PositionDiagramLabelProvider extends CommonLabelProvider {
```

Comment:
De we need this ? because we call `svgParameters.setBusesLegendAdded(false);` when we use this `PositionDiagramLabelProvider`, which will mask BusesLegend ...

Reply:
not a big deal